bootstrapvz/base/manifest-schema.yml

```diff
@@ -45,6 +45,8 @@ properties:
         enum: [i386, amd64, arm64]
       userspace_architecture:
         enum: [i386]
+      grub_enable_timeout: {type: boolean}
+      grub_disable_pnin: {type: boolean}
```

Comment:
Don't you think that instead of using a namespaced option, it would be better if there was a key named `grub` under `bootloader`, where you can set those options? Albeit it is not clear to me what `pnin` would mean, so a bit more verbose name would be nice.

Also, if there's an option to enable the timeout, wouldn't make sense to define the timeout value? I wouldn't expect to set it to `true` don't knowing how much this means.